CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Simplyfied merging logic for backend configs using mergo
```

Comment:
The changes referred by this line, does not seem to be related to monitoring. Unless I missed something, this is a scope creep and have to go into a PR of its own.

Reply:
Point taken. This was an effort to make the abomination called `scheduleRunner.Start()` in the schedule package a bit more manageable. I since created a separate ticket to clean that monster a bit. I'd like to keep this in here as I'm afraid to break stuff (which is very much a red flag that shows how bad that functions is currently.)